test_app/after/src/paper_and_ribbon/indent_18.erl

```diff
@@ -1,9 +1,9 @@
 -module(indent_18).
 
--format #{break_indent => 1,
-          inline_clause_bodies => true,
-          paper => 50,
-          sub_indent => 8}.
+-format #{break_indent => 1}.
```

Comment:
This and other similar changes are due to a subtle bug (fixed in OTP26's `erl_syntax`) that reordered map items when they were found in attributes. I decided to just put the items in the order in which `erl_syntax` returns them (thus, hiding the bug in OTP25) so that tests run seamlessly in 25 and 26.